src/Symfony/Component/VarExporter/Tests/VarExporterTest.php

```diff
@@ -209,6 +210,10 @@ public function provideExport()
         yield ['private-constructor', PrivateConstructor::create('bar')];
 
         yield ['php74-serializable', new Php74Serializable()];
+
+        if (\PHP_VERSION_ID >= 80100) {
+            yield ['unit-enum', FooUnitEnum::Bar, true];
```

Comment:
Let's try with `[FooUnitEnum::Bar]` instead, this should spot that there is a missing check in `Exporter` on the line that looks like:
`} elseif (!\is_object($value) && !$value instanceof \__PHP_Incomplete_Class) {`

Reply:
Done!